src/main/java/org/jitsi/videobridge/BitrateController.java

```diff
@@ -93,18 +113,32 @@
     private final boolean disableLastNNotifications;
 
     /**
+     * A boolean that indicates whether or not we should trust the bandwidth
+     * estimations.
+     */
+    private final boolean trustBwe;
```

Comment:
maybe we could update this comment with the behavior if bwe is not trusted? (looks like we just hard-code it to max?)